src/core/UnstyledLink/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { HTMLAttributes } from 'react';
+import styled from 'styled-components';
+
+import { useLinkContext } from '../../contexts';
+
+export interface UnstyledLinkProps {
+  children: React.ReactNode;
+  className?: string;
+  external?: boolean;
+  to: string;
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => void;
+}
+
+const httpRegExp = /^(http|https):\/\//;
+
+export const UnstyledLink = React.memo(
+  React.forwardRef<HTMLAnchorElement, UnstyledLinkProps>((props, ref) => {
+    const LinkComponent = useLinkContext();
+
+    const { to, external, ...restProps } = props;
+
+    const shouldNewTabOpen = external || props.to.search(httpRegExp) !== -1;
```

Comment:
```suggestion
    const shouldOpenNewTab = external || props.to.search(httpRegExp) !== -1;
```

요렇게 순서를 바꾸면 좀 더 자연스러운 표현이지 않을까... (영알못임)